lib/api/src/externals/function.rs

```diff
@@ -124,7 +129,7 @@ impl Function {
     pub fn new_with_env<F, Env>(store: &Store, ty: &FunctionType, env: Env, func: F) -> Self
     where
         F: Fn(&mut Env, &[Val]) -> Result<Vec<Val>, RuntimeError> + 'static,
-        Env: Sized + 'static,
+        Env: Sized + crate::WasmerEnv + 'static,
```

Comment:
Can't we `use wasmer::WasmerEnv` and replace the code as such:

```suggestion
        Env: Sized + WasmerEnv + 'static,
```

Thanks :-).
```